src/server/visual-diff-plugin.js

```diff
@@ -191,6 +192,9 @@ export function visualDiff({ updateGoldens = false, runSubset = false } = {}) {
 				const page = session.browser.getPage(session.id);
 				await page.screenshot(screenshotOpts);
 
+				// losslessly compress and remove metadata from screenshot
+				if (isCI) execSync(`optipng -quiet -strip all ${screenshotOpts.path}`);
```

Comment:
We can always do this, which would cause everything to fail, or maybe we can do this only if there are differences, and slowly update goldens as they change.

Reply:
Or maybe we can time it with a playwright browser upgrade.